finder-app/finder.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#Checking if the number of arguments is not equal to two
+if [ $# -ne 2 ]
+    then
+        echo "Echo expected two arguments";
```

Comment:
@Sona-Subair think about how you could make this error message more descriptive to tell the user what is required.

Reply:
@dwalkes Thanks for the feedback. I will explain the expected arguments as well in the error message.